config/webpack/postcss/README.md

```diff
@@ -0,0 +1,100 @@
+# Terra Postcss Theme plugin
+
+The purpose of this plugin is to create a default theme from a scoped theme and to remove any known themes that are not desired.
+
+The plugin will read from the same `terra-theme.config.js` that `terra-aggregate-themes` script reads from. Eventually `terra-aggregate-theme`s will be deprecated in favor of this strategy.
+
+## Configuration
+
+### terra-theme.config.js
+
```

Comment:
can we link: https://github.com/cerner/terra-toolkit/pull/390/files#diff-9c1c40ebc66be0d348d616ea7e89ccdfR1?